### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by Blizzard03** [leokhoa/laragon#604] [Ideas]
need dark mode UI
light mode make my eyes hurt for sure 
thx

**Comment by ainxgans** [comment]
How long you see Laragon ? in may case i just click start all and close it.

**Reply by Blizzard03** [reply]
almost 3 hours i guess

**Reply by ainxgans** [reply]
buset bro

**Reply by nflnsr** [reply]
3 jam mantengin laragon (￣﹃￣)

**Comment by thisis-normal** [comment]
I really don't think we need dark mode on laragon

**Reply by Blizzard03** [reply]
Maybe for you, but for me light mode is make my eye burn for sure

**Reply by jesusAG28** [reply]
Just make it start on windows start up, you just need dark mode on your IDE and use right ckick on your taskbar icons for changing php version and more

**Comment by Pen-y-Fan** [comment]
Install the dark reader extension

<img width="1001" alt="image" src="https://github.com/leokhoa/laragon/assets/40126936/bb18dc15-fb35-4ae3-933a-0af0b597cfcd">

**Reply by nachat-ayoub** [reply]
@denisapain I think you are just a dark mode hater at this point.
